lib/v2/jou/utils/index.js

```diff
@@ -0,0 +1,67 @@
+// 导入got库，该库用来请求网页数据
+const got = require('@/utils/got');
+// 导入cheerio库，该库用来解析网页数据
+const cheerio = require('cheerio');
+// 导入parseDate函数，该函数用于日期处理
+const { parseDate } = require('@/utils/parse-date');
+// 导入timezone库，该库用于时区处理
+const timezone = require('@/utils/timezone');
+
+async function getItems(ctx, url, host, tableClass, timeStyleClass, titleStyleClass) {
+    // 发起Http请求，获取网页数据
+    const response = await got({ url, https: { rejectUnauthorized: false } });
+    // 解析网页数据
+    const $ = cheerio.load(response.data);
+
+    // 通知公告的items的标题、url链接、发布日期
+    const list = $(`table.${tableClass} > tbody > tr`)
+        .filter((index, item) => {
+            const currentItem = $(item);
+            const item1 = currentItem.find('td:eq(1)');
+            const hasLink = item1.find('a').length > 0;
+            const isLastItem = index === $(`table.${tableClass} > tbody > tr`).length - 1;
+            return hasLink && !isLastItem;
+        })
+        .toArray()
+        .map((item) => {
+            const currentItem = $(item); // 获取当前tr元素的jQuery对象
+            const item1 = currentItem.find('td:eq(1)'); // 获取当前tr元素下的第二个td
+            const item2 = currentItem.find('td:eq(2)'); // 获取当前tr元素下的第三个td
+            const link = host + item1.find('a').attr('href');
+
+            return {
+                title: item1.find('a').attr('title'),
+                link: link.replace('..', ''),
```

Comment:
Please fix CodeQL failure
```suggestion
            const link = new URL(item1.find('a').attr('href'), url).href;

            return {
                title: item1.find('a').attr('title'),
                link,
```